doc/doxygen/src/ForManagers.dox

```diff
@@ -31,11 +28,14 @@
  *   session state management and other auditing/logging functionality.
  *   It also provides a degree of isolation against future API changes.
  *
- * - The @ref ManagerInterface is designed to be stateless. The
- *   response to any method should only depend on the underlying asset
- *   data and any other objects passed into each call. The same logical
- *   operation may be spread out over time and space. It is critical that
- *   any particular implementation does not rely on local in-memory state.
+ * - Methods of the @ref ManagerInterface are required to be reentrant
+ *   and thread-safe. The response to any method should only depend on
+ *   the local state established during
+ *   @fqref{managerApi.ManagerInterface.initialize} "initialization",
+ *   the underlying asset data and any other objects passed into each
+ *   call. The same logical operation may be spread out over time and
+ *   space. It is critical that any particular implementation does not
+ *   rely on local in-memory state.
```

Comment:
[Consider] I think this could be a cause for confusion. saying that the method mad depend on "local state established during initialization", but then going onto say that an implementation must not depend on "local in-memory state".

This reads to me as though initialize will store state outside of memory, which I don't think is the case. Not sure a better alternative, maybe "does not rely on state created from interactions with any given manager instance." ?

Reply:
Aye.